src/main/java/uk/gov/ons/census/casesvc/service/FulfilmentRequestService.java

```diff
@@ -61,7 +61,7 @@ public void processFulfilmentRequest(ResponseManagementEvent fulfilmentRequest)
     }
   }
 
-  private Case prepareIndividualResponseCaseFromParentCase(Case parentCase) {
+  public Case prepareIndividualResponseCaseFromParentCase(Case parentCase) {
```

Comment:
Move this function into CaseService as it's being used in multiple places

Reply:
done but knock-on effect with tests, out of scope and should be in tech backlog